src/content/docs/en/install/auto.mdx

```diff
@@ -1,154 +1,424 @@
+      yarn create astro --template <github-username>/<github-repo>
+      ```
+      </TabItem>
+    </Tabs>
+
+    By default, this command will use the template repository’s `main` branch. To use a different branch name, pass it as part of the `--template` argument: `<github-username>/<github-repo>#<branch>`.
+
+
+3. Answer the questions and follow the instructions of the CLI wizard.
+
+4. You can now [start the Astro dev server](#start-the-astro-dev-server) and see a live preview of your project while you make it your own!
+</Steps>
+
+### Manual Setup
+
+This guide will walk you through the steps to manually install and configure a new Astro project.
+
+If you prefer not to use our automatic `create astro` CLI tool, you can set up your project yourself by following the guide below.
+
+<Steps>
+1. Create your directory
+
+    Create an empty directory with the name of your project, and then navigate into it.
+
+    ```bash
+    mkdir my-astro-project
+    cd my-astro-project
+    ```
+
+    Once you are in your new directory, create your project `package.json` file. This is how you will manage your project dependencies, including Astro. If you aren't familiar with this file format, run the following command to create one.
+
+    <Tabs>
+      <TabItem label="npm">
+      ```shell
+      npm init --yes
+      ```
+      </TabItem>
+      <TabItem label="pnpm">
+      ```shell
+      pnpm init 
+      ```
+      </TabItem>
+      <TabItem label="yarn">
+      ```shell
+      yarn init --yes
+      ```
+      </TabItem>
+    </Tabs>
+
+
+
+2. Install Astro
+
+    First, install the Astro project dependencies inside your project.
+
+    :::note[Important]
+    Astro must be installed locally, not globally. Make sure you are *not* running `npm install -g astro` `pnpm add -g astro` or `yarn add global astro`.
+    :::
+
+    <Tabs>
+      <TabItem label="npm">
+      ```shell
+      npm install astro
+      ```
+      </TabItem>
+      <TabItem label="pnpm">
+      ```shell
+      pnpm add astro
+      ```
+      </TabItem>
+      <TabItem label="yarn">
+      ```shell
+      yarn add astro
+      ```
+      </TabItem>
+    </Tabs>
+
+    Then, replace any placeholder "scripts" section of your `package.json` with the following:
+
+    ```json title="package.json" del={2} ins={3-6}
+      "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "dev": "astro dev",
+        "start": "astro dev",
+        "build": "astro build",
+        "preview": "astro preview"
+      },
+    ```
+
+    You'll use these scripts later in the guide to start Astro and run its different commands.
+
+3. Create your first page
+
+    In your text editor, create a new file in your directory at `src/pages/index.astro`. This will be your first Astro page in the project.
+
+    For this guide, copy-and-paste the following code snippet (including `---` dashes) into your new file:
```

Comment:
not compound word nit
```suggestion
    For this guide, copy and paste the following code snippet (including `---` dashes) into your new file:
```

Reply:
omg that was NOT me, just copying from the old "manual" text. Thanks for catching that!